pkg/download/jsoncreator/json_creator_test.go

```diff
@@ -163,6 +164,7 @@ func TestReplaceKeyProperties(t *testing.T) {
 
 	assert.Assert(t, data["metadata"] == nil, "metadata should be removed")
 	assert.Assert(t, data["id"] == nil, "id should be removed")
+	assert.Assert(t, data["entityId"] == nil, "id should be removed")
```

Comment:
I would write here (for all) what id should be removed. Here it would be 'entityId', in the field below 'identifier'